notebooks/evaluation/data/0.2.3/synthetic/rag.csv

```diff
@@ -0,0 +1,7 @@
+user_story,id,notes,input,context,expected_output
```

Comment:
sorry to be a PITA but this is one of those things i get weird about, pls could we encode this as just pure python because:

1. csv is a never ending nightmare, I see you are encoding structured objects like lists, this goes wrong all the time
2. you only have seven records.. so there is no significant readability benefit from using CSV
3. pandas adds a lot of bloat, im on a mission to cut this out
4. we can drop your code that turns CSV into python

Reply:
json? or csv-json, i.e.:

```
"name", "age", "address"
"Larry-The-Cat", 6, ["10 Downing Street", "London", "SW1A 2AA"]
```